src/components/table/partial-styles/_row-selection.scss

```diff
@@ -22,35 +22,14 @@ $checkbox-distance-to-left-edge: 0.5rem;
     border-width: 0;
     z-index: $table--limel-table--row-selector;
 
-    text-overflow: unset !important; // Otherwise a `…` will be rendered besides the checkbox, since we truncate all cells
-    padding-left: $checkbox-distance-to-left-edge !important;
-    padding-top: 0.25rem !important;
-
-    limel-checkbox {
-        margin-top: functions.pxToRem(1);
-    }
-
-    .tabulator-row-odd & {
-        background-image: linear-gradient(
-            to right,
-            rgb(var(--table-row-background-color--odd)) 70%,
-            rgb(var(--table-row-background-color--odd), 0)
-        );
-    }
-
-    .tabulator-row-even & {
-        background-image: linear-gradient(
-            to right,
-            rgb(var(--table-row-background-color--even)) 70%,
-            rgb(var(--table-row-background-color--even), 0)
-        );
-    }
-
-    .tabulator-row.active & {
+    &:before {
+        content: '';
+        inset: 0.25rem 0;
+        position: absolute;
```

Comment:
the 3rd commit fixes this issue:

<img width="330" height="163" alt="Image" src="https://github.com/user-attachments/assets/0c7da567-d75e-41f8-8a2f-9cdcbc6dbc7e" />